app/assets/stylesheets/_hint.scss

```diff
@@ -0,0 +1,28 @@
+
+.hint-box {
+	position: relative;
+	margin: 1.5% 0;
+	padding: 1.5%;
+	background-color: $wisp;
+
+	.type {
+		position: absolute;
+		top: 1rem;
+		right: 1rem;
+		color: $gray;
+		font-size: $fs-xsmall;
+	}
+
+	.title {
+		margin-bottom: .2em;
+		font-size: $fs-small;
+		font-weight: $fw-bold;
+	}
+
+	.desc {
+		margin-bottom: 0;
+		font-size: $fs-small;
+	}
+}
+
```

Comment:
dude wtf is up with this spacing? Very not-:rainbow:

Honestly though, I kinda don't care what is in the scss though so if you prefer whatever atrocity this is, I can adjust my styles editor to not freak out at this stuff.